test/erlfmt_SUITE.erl

```diff
@@ -1053,18 +1083,18 @@ contains_pragma(Config) when is_list(Config) ->
     ?assertEqual(
         true,
         erlfmt:contains_pragma_string(
```

Comment:
Looking at those tests, I'm not really sure about their value. In particular we don't use `erlfmt:contains_pragma_string` in regular code, so this seems like it's testing a function exported only for tests

Reply:
contains_pragma_string is calling `read_nodes(erlfmt_scan:string_node(String), FileName, require)` much like the regular code is doing.  At least this is the idea.